pkg/api/apis.go

```diff
@@ -0,0 +1,102 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+import (
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/maps"
+)
+
+type APIs map[string]API
+
+func NewAPIs() APIs {
+	return newAPIs(configEndpoints)
+}
+
+func NewV1APIs() APIs {
+	return newAPIs(configEndpointsV1)
+}
+
+func newAPIs(t []API) APIs {
+	apis := make(APIs)
```

Comment:
You can initialize this already with len(t)

Reply:
does thant optimization really makes any benefit here?
Downside is that it makes code less readable